src/main/java/vazkii/botania/common/crafting/ModCraftingRecipes.java

```diff
@@ -2318,9 +2318,9 @@ private static void addPane(Block block, Block pane) {
 	}
 
 	private static IRecipe addQuartzRecipes(int meta, Item req, Block block, Block stairs, Block slab) {
-		GameRegistry.addRecipe(new ItemStack(block),
+		GameRegistry.addRecipe(new ShapedOreRecipe(new ItemStack(block),
 				"QQ", "QQ",
-				'Q', new ItemStack(ModItems.quartz, 1, meta));
+				Character.valueOf('Q'), LibOreDict.QUARTZ[meta]));
```

Comment:
That was because of a dumb mistake of mine, I forgot I needed to make it a shaped recipe at first >.> and was getting null pointer errors, and was used to using Character.valueOf() in scala because it won't seem to work in scala without